aksel.nav.no/website/components/website-modules/utils/amplitude.tsx

```diff
@@ -98,9 +108,44 @@ export const usePageView = (router: Router, pageProps: any) => {
       side: window.location.pathname,
       prosent: scrollD,
     });
-  }, []);
+
+    try {
+      if (!(isDevelopment || isTest || isPreview()) && hasMetrics) {
+        const { metrics } = pageProps.page;
+        const _id = pageProps?.page._id;
+        fetch(
+          `/api/log-scroll?id=${_id}&current=${
+            metrics.avgScrollLength || 0
```

Comment:
Ønsket helst at siden ikke skal måtte hente ut metrics for at dette skal fungere. Eksponerer da all metrikk-data ut til brukere på Aksel, noe vi sikkert vil unngå. Tenker kanskje vi bare kan gjøre et enkelt groq-query i api-funksjonen som henter ut metrikk-dataen siden vi får dokument-id herfra?